app/src/main/java/org/thoughtcrime/securesms/database/model/ThreadRecord.java

```diff
@@ -73,7 +75,7 @@ public ThreadRecord(@NonNull String body, @Nullable Uri snippetUri,
   @Override
   public SpannableString getDisplayBody(@NonNull Context context) {
     Recipient recipient = getRecipient();
-    if (isGroupUpdate()) {
+    if (isGroupUpdate() || isGroupUpdateMessage()) {
```

Comment:
What's still using `isGroupUpdate` at this point?

Reply:
The group updates messages prior to the release of that update will still need that test. We can get rid of it eventually, but for now it better to keep it.